.github/workflows/build-master.yml

```diff
@@ -0,0 +1,25 @@
+name: Build master branch of Ballerina datamapper compiler extension
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build on Ubuntu
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Build with Gradle
+        env:
```

Comment:
We will also have to add the following [line](https://github.com/ballerina-platform/module-ballerina-docker/blob/master/.github/workflows/daily-build.yml#L22).

```
GITHUB_TOKEN: ${{ secrets.BALLERINA_STDLIB_TOKEN }}
```
That is required to pull the stdlib artifacts from github.

Reply:
Added this line

Reply:
Ok